salt/modules/systemd.py

```diff
@@ -158,6 +160,123 @@ def systemctl_reload():
     return retcode == 0
 
 
+def _default_runlevel():
```

Comment:
Ahh, I see, I was only looking at the changes for this commit. Didn't realize that code came from another module. Well, if no one is breaking their systems in ways that cause this code to fail, then it surely will always work just fine.